packages/python/tests/conftest.py

```diff
@@ -15,9 +15,9 @@
 from armonik.protogen.worker.agent_service_pb2_grpc import AgentStub
 from typing import List, Union
 
-ca_cert = os.getenv("Grpc__CaCert")
-client_cert = os.getenv("Grpc__ClientCert")
-client_key = os.getenv("Grpc__ClientKey")
+ca_cert = os.getenv("Grpc__CaCert") or None
```

Comment:
Why don't you check GrpcClient__CaCert?
What is the purpose of `or None`?

Reply:
Forgot the Client_CaCert
The "or None" is to force None if the environement variable is set but empty